code/modules/paperwork/folders.dm

```diff
@@ -6,11 +6,46 @@
 	w_class = WEIGHT_CLASS_SMALL
 	pressure_resistance = 2
 	resistance_flags = FLAMMABLE
+	var/component_type = /datum/component/storage/concrete

```

Comment:
as far as i can tell this var isnt needed, if you want to do different parameters you can use the duplicate behaviour.